### DISCUSSION TITLE
Tasmota knx groupread config not working

### DISCUSSION THREAD

**Discussion by WoutervanZijl** [ascillato/Tasmota_KNX#594] [Q&A]
Hello there,

I can not seem to understand how groupread request gets to work.
I have used different option.

My listen group adres is 2/1/56
Then the group read adres would be the same as i have read from the text. Tried several things. 
Other knx device do get read request and works perfect but those are real knx devices like thermostat.

Only my shelly tasmota is not responding to it. I can read on the bus a request from home assistant but it does not respond to it.

Anybody has a working snapshot of the knx configuration page that work with read request?

Regards 

Wouter van Zijl

**Comment by Noschvie** [comment]
Do you have Multicast enabled on your WIFI ?

**Reply by WoutervanZijl** [reply]
Yes because i can communicate with different tasmotas. I am using sonof rf bridge to send knx adresses and everything responds normal. Only not group read.

**Reply by WoutervanZijl** [reply]
My access point is changed to a ruckus r720 instead of the cisco one and also a draytek router to get better control.
